<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterWindow</class>
 <widget class="QDialog" name="RegisterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="username">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>username: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="usernamelinedit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Password">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Password: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordeditline">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="RetypePassword">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Retype Password:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditRetypePassword">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="dateofbirth">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Date of birth:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="monthcombobox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>January</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>February</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>March</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>April</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>May</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>June</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>July</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>August</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>September</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>October</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>November</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>December</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="monthlabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>170</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Month: </string>
   </property>
  </widget>
  <widget class="QLabel" name="daylabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>170</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>day:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="daylineedit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>190</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="yearlabel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>170</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Year: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="yearlineedit">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>190</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="gendergroupbox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Gender: </string>
   </property>
   <widget class="QRadioButton" name="maleradiobutton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Male</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="femaleradioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Female</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="accounttypegroupBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>240</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Account type: </string>
   </property>
   <widget class="QRadioButton" name="UserradioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>User</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="adminradioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>admin</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="FavoriteGenreGroupBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>330</y>
     <width>271</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Favorite Genre</string>
   </property>
   <widget class="QCheckBox" name="actioncheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>76</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Action</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="DramacheckBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>76</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Drama</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="comedycheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>76</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Comedy</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="horrorcheckBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>76</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Horror</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="romancecheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>76</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Romance</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="othercheckBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>76</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Other</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="RegisterpushButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>460</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </widget>
  <widget class="QLabel" name="alreadyexistingerrorlabel">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>40</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;*Already existing&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="notmatchingerrorlabel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>100</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;*Not Matching&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="agelesstan12errorlabel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>150</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;*Age is less than 12&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="fieldsnotfielderrorlabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>460</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;* All fields must be filled&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
